src/main/kotlin/org/jitsi/nlj/dtls/DtlsConfig.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright @ 2019 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.nlj.dtls
+
+import org.jitsi.config.JitsiConfig
+import org.jitsi.metaconfig.config
+import java.time.Duration
+
+class DtlsConfig {
+    companion object {
+        val handshakeTimeout: Duration by config {
```

Comment:
Can we make this a normal class instead of a static val?  In general it's nice to avoid the static access for these because it makes testing harder.  The Client/Server impl would then each declare a member of `DtlsConfig`